.gitignore

```diff
@@ -11,6 +11,9 @@ pids
 *.seed
 *.pid.lock
 
+#remove js files
```

Comment:
I see changes in js files here, should I ignore them? Should we remove the js files that are already included?

Reply:
I've removed the js files